packages/backend/src/lib/daoHausClient.ts

```diff
@@ -1,8 +1,15 @@
 import { GraphQLClient } from 'graphql-request';
 
 import { CONFIG } from '../config';
-import { getSdk } from './autogen/daohaus-sdk';
+import { getSdk, Sdk } from './autogen/daohaus-sdk';
 
-export const daoHausClient = getSdk(
-  new GraphQLClient(CONFIG.daoHausGraphqlURL),
-);
+export function clientFactory(chain: string): Sdk {
+  switch (chain) {
+    case 'polygon':
+      return getSdk(new GraphQLClient(CONFIG.daoHausPolygonGraphqlURL));
```

Comment:
I'd suggest caching this locally. Otherwise a new GraphQLClient is created on every api call.

Reply:
Good catch! I have updated that now.